build/deps.mk

```diff
@@ -8,7 +8,7 @@ deps: tools deps-only
 
 deps-only:
 	@echo "=== $(PROJECT_NAME) === [ deps             ]: Installing package dependencies required by the project..."
-	@$(GO) mod tidy
+	#@$(GO) mod tidy
```

Comment:
Should we change this back?

Reply:
`go mod tidy` is broken with an update we made to goreleaser recently.  It's the same issue we saw in the kubernetes operator project that required indirect refs to `go-autorest`.